packages/backend-defaults/report-urlReader.api.md

```diff
@@ -316,6 +316,7 @@ export class GithubUrlReader implements UrlReaderService {
       treeResponseFactory: ReadTreeResponseFactory;
       credentialsProvider: GithubCredentialsProvider;
     },
+    logger?: LoggerService | undefined,
```

Comment:
Can you move this into `deps` instead? This is a common two-argument form that we use a lot, with one primary thing and the rest in an options object